src/util/window.js

```diff
@@ -82,7 +81,7 @@ function restore(): Window {
     window.ImageData = window.ImageData || function() { return false; };
     window.ImageBitmap = window.ImageBitmap || function() { return false; };
     window.WebGLFramebuffer = window.WebGLFramebuffer || Object;
-    extend(_window, window);
+    Object.assign(_window, window); // eslint-disable-line no-restricted-properties
```

Comment:
This will break in IE11. I misunderstood what the bublé `objectAssign` configuration does.

Reply:
It won't because this file is a Node stub for testing, replaced by `util/browser/window.js`.

Reply:
🤕